pkg/api/v1/types.go

```diff
@@ -2292,6 +2292,12 @@ type Binding struct {
 	Target ObjectReference `json:"target"`
 }
 
+// Preconditions must be fulfilled before an operation (update, delete, etc.) is carried out.
+type Preconditions struct {
+	// Specifies the target UID.
+	UID *types.UID `json:"uid,omitempty"`
```

Comment:
Yeah, we do check RV in the Update path, but that RV is extracted from the new object sent by the user, not from an "UpdateOptions". As lavalamp commented, it should be straight-forward if we want to add RV to Preconditions later :)